docs/topics/mta-web-config-jira-credentials.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-jira-credentials_{context}"]
+= Configuring Jira credentials
+
+To define a Jira instance in {ProductShortName} and establish a connection to that instance, you must create an {ProductShortName} credential to authenticate to the Jira instance's API.
+
+Two types of credentials are available:
+
+* *Basic auth* - for Jira Cloud and a private Jira Server or Datacenter
+* *Bearer Token* - for a private Jira Server or Datacenter
+
+.Procedure
+
+. In *Administration* view, click *Credentials*.
++
+The *Credentials* page opens.
+. Click *Create new*.
+. Enter the following information:
+    * Name
+    * Description (optional)
+. In the *Type* list, select *Basic Auth (Jira)* or *Token Bearer (Jira)*:
++
+* If you have selected *Basic Auth (Jira)*, proceed as follows:
+
```

Comment:
"if you select.." or "If you selected..." . I recommend the first, but either way, please use the simple form of the verb.